unicore/hub/service/locale/messages.pot

```diff
@@ -0,0 +1,160 @@
+msgid "${username} is not unique"
+msgstr ""
+
+#: unicore/hub/service/validators.py:72 unicore/hub/service/validators.py:84
+msgid "${data} is not a dictionary"
+msgstr ""
+
+#: unicore/hub/service/validators.py:78
+msgid "${key} is not a valid app UUID"
+msgstr ""
+
+#: unicore/hub/service/validators.py:93
+msgid "${pin} is not ${length} digits long"
+msgstr ""
+
+#: unicore/hub/service/validators.py:100
+msgid "${pin} contains non-digit characters"
+msgstr ""
+
+#: unicore/hub/service/sso/views.py:36
+msgid "Username"
+msgstr ""
+
+#: unicore/hub/service/sso/views.py:40
+msgid "Password"
+msgstr ""
+
+#: unicore/hub/service/sso/views.py:65
+msgid "Username or password is incorrect"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/base.jinja2:18
+#: unicore/hub/service/sso/templates/logout_success.jinja2:3
+msgid "Sign Out"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/base.jinja2:20
+#: unicore/hub/service/sso/templates/join.jinja2:35
+#: unicore/hub/service/sso/templates/login.jinja2:4
+#: unicore/hub/service/sso/templates/login.jinja2:16
+msgid "Sign In"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/base.jinja2:30
+msgid "About"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/base.jinja2:31
+msgid "Contact"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/base.jinja2:32
+msgid "Terms"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/join.jinja2:4
+#: unicore/hub/service/sso/templates/join.jinja2:9
+#: unicore/hub/service/sso/templates/join.jinja2:31
+#: unicore/hub/service/sso/templates/login.jinja2:20
+msgid "Join"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/join.jinja2:4
+#: unicore/hub/service/sso/templates/login.jinja2:4
+#: unicore/hub/service/sso/templates/logout_success.jinja2:3
+msgid "Universal Core"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/join.jinja2:10
+msgid "You will use these details to sign in in future."
+msgstr ""
+
+#: unicore/hub/service/sso/templates/join.jinja2:17
+msgid "Choose a Username"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/join.jinja2:21
+#, python-format
+msgid ""
+"%(username)s is already taken. Not you? Please choose a different "
+"username."
+msgstr ""
+
+#: unicore/hub/service/sso/templates/join.jinja2:22
+#, python-format
+msgid "Is this you? <a href=\"%(login_url)s\">Sign in here.</a>"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/join.jinja2:29
+msgid "Choose a 4 Digit PIN (E.g 4265)"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/join.jinja2:34
+msgid "Aready joined?"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/login.jinja2:14
+msgid "4 Digit PIN"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/login.jinja2:19
+msgid "Don't have an account?"
+msgstr ""
+
+#: unicore/hub/service/sso/templates/login.jinja2:24
+#, python-format
+msgid "You are signed in as %(username)s."
+msgstr ""
+
+#: unicore/hub/service/sso/templates/login.jinja2:25
+#, python-format
+msgid "Click <a href=\"%(logout_url)s\">here</a> to sign out."
```

Comment:
@Rizziepit the link should not be here. You'll have to find a way around this. Possibly need to change the link format.